pallets/collator-assignment/src/lib.rs

```diff
@@ -285,6 +287,30 @@ pub mod pallet {
                 }
             };
 
+            // TODO: this probably is asking for a refactor
+            // only apply the onCollatorAssignedHook if sufficient collators
+            for para_id in &container_chain_ids {
```

Comment:
Let's try to do this in a cleaner manner

Reply:
Yes but we can do it later, when parathreads and parachains are charged differently